coldfront/core/project/tests/test_views/test_project_removal_request_views.py

```diff
@@ -0,0 +1,650 @@
+from django.test import TestCase
```

Comment:
Can you test that the "Cluster Account Status" gets set to "Denied" and that the `AllocationUserStatusChoice` gets set to "Removed"?

This'll involve creating those allocation-related objects, which you may not have done before. See [here](https://github.com/ucb-rit/coldfront/pull/307/files#diff-7a5e4b8bc3fd0a15ae3e3a16913e7747ca7365a3999a83da77a7ab4b6336da4cR17) for an example.

(Long-term, I should add some test methods to make setting up these objects easier.)